projects/BFDR/IJEBirth.cs

```diff
@@ -11,12 +11,12 @@ namespace BFDR
     /// FHIR based <c>BirthRecord</c> to the IJE format for a specific field, and
     /// the setters convert from IJE format for a specific field and set that value
     /// on the embedded FHIR based <c>BirthRecord</c>.</summary>
-    public class IJEBirth : IJE
+    public class IJEBirth : IJENatality
     {
         private readonly BirthRecord record;
 
         /// <summary>Constructor that takes a <c>BirthRecord</c>.</summary>
-        public IJEBirth(BirthRecord record, bool validate = true)
+        public IJEBirth(BirthRecord record, bool validate = true) : base(record, validate)
```

Comment:
I think the contents of this Constructor could be moved to the base constructor in IJE Natality.